src/api/task/api/dto/task.go

```diff
@@ -1,51 +1,62 @@
 package dto
 
 import (
+	"proyectos/src/api/constants"
+	"proyectos/src/api/errors"
 	"proyectos/src/api/task/domain/model"
 	"time"
 )
 
 // Task dto
 type Task struct {
 	ID             int64     `validate:"gt=0" json:"id"`
-	Name           string    `validate:"required,min=2,max=50" json:"name"`
+	Name           string    `validate:"required,min=2,max=500" json:"name"`
 	Description    string    `validate:"required" json:"description"`
 	StartDate      time.Time `validate:"required" json:"start_date"`
-	HoursWorked    int       `validate:"required" json:"hours_worked"`
+	WorkedHours    int       `validate:"required" json:"worked_hours"`
 	EstimatedHours int       `validate:"required" json:"estimated_hours"`
-	ProjectId      int64     `validate:"required" json:"project_id"`
+	ProjectID      int64     `validate:"required" json:"project_id"`
 	State          string    `validate:"required" json:"state"`
 	CreationDate   time.Time `validate:"required" json:"creation_date"`
```

Comment:
Why is CreationDate a required field, shouldn't this be optional and simply enforce it's initialization with sysdate when we create a task?

Reply:
@AlexDaciuk @julitaras qué opinan?

Reply:
por cuestiones de auditoria tendria que tener un CreationDate, me parece bien que se le planche systime al momento de crearse y listo